lib/src/puppetdb-cli.cc

```diff
@@ -25,7 +39,7 @@ namespace logging = leatherman::logging;
 string
 version()
 {
-    LOG_DEBUG("puppetdb-cli version is %1%", PUPPETDB_CLI_VERSION_WITH_COMMIT);
+    LOG_DEBUG("puppetdb-cli version is {1}", PUPPETDB_CLI_VERSION_WITH_COMMIT);
```

Comment:
This is actually broken currently, noticed it when adding another logging error. Thought I would fix it up here.